crates/ruff_python_formatter/src/expression/mod.rs

```diff
@@ -66,10 +65,15 @@ impl FormatRule<Expr, PyFormatContext<'_>> for FormatExpr {
     fn fmt(&self, expression: &Expr, f: &mut PyFormatter) -> FormatResult<()> {
         let parentheses = self.parentheses;
 
+        // Nested expressions that are in some outer expression's parentheses may be formatted in
+        // fluent style
+        let fluent_style = f.context().node_level() == NodeLevel::ParenthesizedExpression
+            && is_fluent_style_call_chain(expression, f.context().source());
```

Comment:
I spent way to much time trying to come up with something better, but i'm afraid the answer is yes. I think the node level is the biggest problem here because e.g. just inlining helpers doesn't seem to work.